app/serializers/organization_serializer.rb

```diff
@@ -1,26 +1,37 @@
 class OrganizationSerializer
   include RestPack::Serializer
   include ContentSerializer
+  include MediaLinksSerializer
 
-  attributes :id, :name, :display_name, :description, :introduction, :title, :href
+  attributes :id, :name, :display_name, :description, :introduction, :title, :href, :avatar_src, :background
```

Comment:
Cards are on my list of things to add, so I thought I'd include this while I was adding avatars.